webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9158,6 +9187,31 @@ var userOverride = {
                                 : {}
                         );
                     }
+                } else if (
+                    !(
+                        msg.metadata.obsolete && msg.metadata.obsolete == 'true'
+                    ) &&
+                    msg.metadata.KM_FIELD
```

Comment:
Looks like we are not going to give it a template id. 
I think we should associate a rich message with some template id.

Reply:
in autosuggestion we don't have a template Id, not sure if we need to give it here?
@prochnost04 when do we give templateId and when not?

Reply:
Sir for all richmessages we have provided template id except for KM_AUTO_SUGGESTION

Not sure why the team didn't added any template id for KM_AUTO_SUGGESTION
I think we should have a consistency across our rich messages and that's the reason I think we should provide one.
Providing template id will make sure that we don't have to add exception in other platform (dashboard, api server, etc) when rich messages are processed.

Reply:
@aryankirsali please add in that case